packages/angular/src/generators/ng-add/migrators/migrator.ts

```diff
@@ -27,6 +33,52 @@ export abstract class Migrator {
   abstract migrate(): Promise<void> | void;
   abstract validate(): ValidationResult;
 
+  protected convertAsset(asset: string | any): string | any {
+    if (typeof asset === 'string') {
+      return this.convertSourceRootPath(asset);
+    } else {
```

Comment:
Not so much a worry _right now_ but `polyfills` for Angular 15 will be `string | string[]` so we may need to handle that

Reply:
It would make sense to cover that in the Angular 15 branch.